blender/arm/make_renderpath.py

```diff
@@ -307,8 +308,13 @@ def build():
             assets.add_khafile_def('rp_ssr')
             assets.add_shader_pass('ssr_pass')
             assets.add_shader_pass('blur_adaptive_pass')
-            if rpdat.arm_ssr_half_res:
-                assets.add_khafile_def('rp_ssr_half')
```

Comment:
Is this removed by purpose? It looks as if this define isn't set anywhere else so if half res is enabled in the Blender UI the define will not be set.

Maybe this is related to the problem with half res that you've mentioned?

Reply:
Ok that's fixed.